Sources/PathKit.swift

```diff
@@ -152,8 +152,15 @@ extension Path {
   ///
   public func abbreviate() -> Path {
 #if os(Linux)
-    // TODO: actually de-normalize the path
-    return self
+    let env = ProcessInfo.processInfo.environment
+    guard let home = env["HOME"], self.path.contains(home) else { return self }
```

Comment:
`contains`? What if the path was something like `/mnt/backups/home/kyle/test`, this would incorrectly match when `HOME` is set to `/home/kyle/` and the method would return `~/mnt/backupstest`